integration/util/substrate.js

```diff
@@ -42,6 +43,11 @@ function waitForEvent(api, pallet, method, opts = {}) {
 
   let resolve, reject;
   let promise = new Promise((resolve_, reject_) => {
+    if (opts.timeout) {
```

Comment:
This timeout should be helpful in stopping "forever running" tests.

Reply:
yes this helped me a lot thank you